portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -13,10 +14,29 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
```

Comment:
The servlet is returning a JSON so it would be more correct to have "application/json". The content type is used so that the receiver (in our case your JavaScript) can know how to parse the response. We don't need it that much (this field could be anything), but since we're returning a JSON, we should also reflect it in the content type.